### ISSUE TITLE
When mapped, WordPress uploads are almost always 4.1K

### ISSUE THREAD

**Issue by binarykitchen** [getflywheel/local-addon-volumes#14]
On Mac OS using latest Local by Flywheel in conjunction with this local-addon-volumes almost all uploads in WordPress appear to be 4.1K large. Every time. When it’s a video, it can’t be played.

Here a directory listing as an example:

```
/Users/michael-heuberger/Local Sites/(my test site)/app/public/wp-content/uploads/2018/07 
$ la                                                                                                                                                                           
-rw-r--r--  1 michael-heuberger  staff   4.1K 16 Jul 18:23 148863337136.webm
-rw-r--r--  1 michael-heuberger  staff   4.1K 16 Jul 18:20 279892931352.webm
-rw-r--r--  1 michael-heuberger  staff   4.1K 16 Jul 18:22 330014371325.webm
-rw-r--r--  1 michael-heuberger  staff    39K 16 Jul 18:25 145163372442.webm <--- here a working one
-rw-r--r--  1 michael-heuberger  staff   4.1K 16 Jul 18:24 57609106783.webm
```

When you research you will see that symlinks are always 4.1K. But wait, you see one of the above video files has a reasonable size of 39K and can be played. All others are 4.1K

Probably a bug when it comes to Volumes mapping, Docker, Virtual Stuff or so? Not sure.

Any clues?

Thanks
Michael

**Comment by claygriffiths** [comment]
That's interesting, especially considering that's an `la` from your host machine and not inside the container.

If you move the file out of the Local site's folder, are you able to play the `webm`?

**Comment by binarykitchen** [comment]
Nope, even when I move out the bad ones, they can't be played @claygriffiths 

Can you investigate this? It's breaking my wordpress plugin ...

**Comment by binarykitchen** [comment]
Ping? @claygriffiths

**Comment by claygriffiths** [comment]
@binarykitchen,

Got it. So based off of that, this issue is likely occurring when you upload the file and not after the fact. If you upload new webm's do they have a size of `4.1K`?

Do you have Faster Docker Volumes enabled under Preferences » Advanced?

**Comment by binarykitchen** [comment]
Yes I have Faster Docker Volumes enabled. Does this cause this problem? @claygriffiths
